src/LayerEditor/data/layer_geometry_serializer.py

```diff
@@ -58,7 +58,7 @@ def set_default_data(self):
 
     def set_default(self):
         default_regions = [                                                                                # Stratum layer
-            Region(dict(color="##", name="NONE", not_used=True, dim=RegionDim.none))         # TopologyDim.polygon
+            Region(dict(color="gray", name="NONE", not_used=True, dim=RegionDim.none))         # TopologyDim.polygon
```

Comment:
Qt makes the translation. "##" was translated as invalid color (unless I missed some detail in documentation) and therefore default color (black) was used. Genie uses gray instead of black for default region, so I unified the color coding.